tutorials/nservicebus-101/lesson-1/tutorial.md

```diff
@@ -1,18 +1,27 @@
 ---
 title: "NServiceBus 101 Lesson 1: Getting started"
-reviewed: 2016-11-16
+reviewed: 2017-01-26
 ---
 
 In the next 10 minutes, you will learn how to set up a new development machine for NServiceBus and create your very first messaging endpoint.
 
 
-## Prerequisites
+## Before we get started
 
-There are a few things we need in order to build a solution with the latest version of NServiceBus.
+NServiceBus has very few prerequisites. It's very easy to get started.
 
-* This course uses Visual Studio 2015 and .NET Framework 4.6.1, although [NServiceBus only requires .NET Framework 4.5.2](/nservicebus/operations/dotnet-framework-version-requirements.md).
-* NServiceBus requires queuing infrastructure (a [transport](/nservicebus/transports/)) to move messages around. This course uses Microsoft Message Queuing (MSMQ). The [Particular Platform Installer](/platform/installer/) will install MSMQ and its dependencies for you. [Download and run the Platform Installer](/platform/installer/) to get started, or [follow these instructions to configure MSMQ manually](/nservicebus/msmq/#nservicebus-configuration).
+Although [NServiceBus only requires .NET Framework 4.5.2](/nservicebus/operations/dotnet-framework-version-requirements.md), this course uses Visual Studio 2015 and .NET Framework 4.6.1, which includes some useful async-related APIs.
 
+NServiceBus needs queuing infrastructure (a [transport](/nservicebus/transports/)) to move messages around. By default, this course will use **Microsoft Message Queuing (MSMQ)**, which is included with every version of Windows, although it is not installed by default.
+
+NOTE: If MSMQ is not an option for your environment, the [SQL Server Transport](/nservicebus/sqlserver/) can be used as a message transport instead. To get started using SQL Server instead of MSMQ, review the [SQL Server transport setup instructions](../using-sql-transport.md). Throughout the rest of the guide, instructions that differ between the SQL and MSMQ transports will be highlighted in an informational box like this one.
```

Comment:
I have said this before. but i think it is wrong to confuse their first interaction with NSB to conflate transport selction